openmc/data/neutron.py

```diff
@@ -0,0 +1,926 @@
+                    continue
+
+                # Create redundant reaction with appropriate cross section
+                mts = data.get_reaction_components(mt)
+                if len(mts) == 0:
+                    warn('Photon production is present for MT={} but no '
+                         'reaction components exist.'.format(mt))
+                    continue
+
+                # Determine redundant cross section
+                rx = data._get_redundant_reaction(mt, mts)
+                rx.products += _get_photon_products_ace(ace, rx)
+                data.reactions[mt] = rx
+
+        # For transmutation reactions, sometimes only individual levels are
+        # present in an ACE file, e.g. MT=600-649 instead of the summation
+        # MT=103. In this case, if a user wants to tally (n,p), OpenMC doesn't
+        # know about the total cross section. Here, we explicitly create a
+        # redundant reaction for this purpose.
+        for mt in (16, 103, 104, 105, 106, 107):
+            if mt not in data:
+                # Determine if any individual levels are present
+                mts = data.get_reaction_components(mt)
+                if len(mts) == 0:
+                    continue
+
+                # Determine redundant cross section
+                rx = data._get_redundant_reaction(mt, mts)
+                data.reactions[mt] = rx
+
+        # Make sure redundant cross sections that are present in an ACE file get
+        # marked as such
+        for rx in data:
+            mts = data.get_reaction_components(rx.mt)
+            if mts != [rx.mt]:
+                rx.redundant = True
+            if rx.mt in (203, 204, 205, 206, 207, 444):
+                rx.redundant = True
+
+        # Read unresolved resonance probability tables
+        urr = ProbabilityTables.from_ace(ace)
+        if urr is not None:
+            data.urr[strT] = urr
+
+        return data
+
+    @classmethod
+    def from_endf(cls, ev_or_filename, covariance=False):
+        """Generate incident neutron continuous-energy data from an ENDF evaluation
+
+        Parameters
+        ----------
+        ev_or_filename : openmc.data.endf.Evaluation or str
+            ENDF evaluation to read from. If given as a string, it is assumed to
+            be the filename for the ENDF file.
+
+        covariance : bool
+            Flag to indicate whether or not covariance data from File 32 should be
+            retrieved
+
+        Returns
+        -------
+        openmc.data.IncidentNeutron
+            Incident neutron continuous-energy data
+
+        """
+        if isinstance(ev_or_filename, Evaluation):
+            ev = ev_or_filename
+        else:
+            ev = Evaluation(ev_or_filename)
+
+        atomic_number = ev.target['atomic_number']
+        mass_number = ev.target['mass_number']
+        metastable = ev.target['isomeric_state']
+        atomic_weight_ratio = ev.target['mass']
```

Comment:
Question: Just want to confirm here that `ev.target['mass']` represents relative atomic mass or another number equivalent to "atomic weight ratio".